src/algebra/ordered_group.lean

```diff
@@ -817,6 +817,10 @@ abs_le.2 ⟨(neg_add (abs a) (abs b)).symm ▸
   add_le_add (neg_le.2 $ neg_le_abs_self _) (neg_le.2 $ neg_le_abs_self _),
   add_le_add (le_abs_self _) (le_abs_self _)⟩
 
+theorem abs_sub_le_abs_add_abs (a b : α) :
+  abs (a - b) ≤ abs a + abs b :=
```

Comment:
```suggestion
theorem abs_sub (a b : α) : abs (a - b) ≤ abs a + abs b :=
```
makes this match the naming of `abs_add`. Is this name already taken?

Reply:
Yeah... `abs_sub (a b : α) : abs (a - b) = abs (b - a)`. Maybe this one could get renamed to `abs_sub_symm` or `abs_sub_comm`?

Reply:
+1 for `abs_sub_comm`

Reply:
I would also have renamed them `abs_add_le` and `abs_sub_le`, but `abs_sub_le` is also already taken.